assets/queries/k8s/cpu_limits/query.rego

```diff
@@ -0,0 +1,71 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    specInfo := getSpecInfo(document)
+    metadata := document.metadata
+
+    containers := document.spec.containers
+    
+    
+    object.get(containers[index]["resources"]["limits"], "cpu", "undefined") == "undefined"
+ 
+    result := {
+                "documentId":        input.document[i].id,
+                "issueType":         "MissingAttribute",
+                "searchKey":         sprintf("metadata.name=%s.%s.containers.name=%s.resources.limits", [metadata.name, specInfo.path, containers[index].name]),
+                "keyExpectedValue":  sprintf("%s.containers.name=%s does have CPU limits",[specInfo.path, containers[index].name]),
```

Comment:
Perhaps it's clearer to specify the path where the information should be added, like in this case:
sprintf("'%s.containers[%s].resources.limits.cpu' is set",[specInfo.path, index])